packages/tools/src/tools/annotation/LivewireContourTool.ts

```diff
@@ -111,20 +114,19 @@ class LivewireContourTool extends AnnotationTool {
     const FrameOfReferenceUID = viewport.getFrameOfReferenceUID();
     const defaultActor = viewport.getDefaultActor();
 
-    if (!defaultActor || !csUtils.isImageActor(defaultActor)) {
-      throw new Error('Default actor must be an image actor');
-    }
+    // if (!defaultActor || !csUtils.isImageActor(defaultActor)) {
+    //   throw new Error('Default actor must be an image actor');
+    // }
```

Comment:
not sure why you are deleting this, since even stack viewport is imageActor

Reply:
It throws on VideoViewport as that doesn't have a default actor (yet).